_posts/2015-06-07-cosmic-neutrinos.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Improved limits in search for ultrahigh-energy cosmic neutrinos
+author: RaoOfPhysics
+---
+
+_Pierre Auger Observatory finds no neutrino candidates with energies of greater than 1 EeV in data collected between Jan 2004 and Jun 2013_
+
+We are all bathed in a constant stream of particles knows as cosmic rays[^1], which are produced mostly outside the Solar System.
+These high-energy particles were initially suspected to have Earthly origins, but Victor Hess demonstrated in 1912 that they arrive on our planet from non-terrestrial sources.
+The enigmatic nature of cosmic rays has captivated scientists for decades.
+
```

Comment:
To trace where cosmic rays come from, it is useful to look at their interactions with other particle. During these interactions, the cosmic rays lose energy.